### ISSUE TITLE
Struggling to interpret discr_si 

### ISSUE THREAD

**Issue by tim-laurence-dhsc** [mrc-ide/EpiEstim#142]
Hello!

Thank you for open sourcing such a brilliant package, it's a brilliant contribution to the community!

I was trying to better understand the R estimation methods in the package and cannot fully understand the [discr_si](https://github.com/mrc-ide/EpiEstim/blob/master/R/discr_si.R) function.

In lines 62 and 63 it looks like the method of moments for the gamma distribution is being used to find the shape and scale parameter, but shouldn't line 62 be (mu - 1)**2 unless I'm misinterpreting the formula (apologies if I am). https://en.wikipedia.org/wiki/Gamma_distribution

I assume the mu-1 is to make it a shifted gamma - but the only other reference to shifted gamma's I've seen in this context is epidemia where the shifting seems to happen outside the gamma distribution. https://imperialcollegelondon.github.io/epidemia/articles/priors.html

Any references you can link to the shifted gamma formulation to explain the (mu-1) and lines 67-71 would be much appreciated!

Thanks,

Tim

**Comment by jstockwin** [comment]
Hi @tim-laurence-dhsc,

First of all a disclaimer - it's been quite a while since I've worked on EpiEstim, so please do check everything I say yourself as I could be wrong here. These days I am a programmer, not a mathemetician.

You're correct that `discr_si` uses a shifted gamma. This is stated at the top of the file:
```
#########################################################
# Discretized serial interval (assuming a shifted gamma #
# distribution (with shift 1)                           #
#########################################################

#' Discretized Generation Time Distribution Assuming A Shifted Gamma 
#' Distribution
#'
#' \code{discr_si} computes the discrete distribution of the serial interval, 
#' assuming that the serial interval is shifted Gamma distributed, with shift 1.
```

I believe I am correct in saying this is because the methodology works by looking at cases each day. The Serial Interval (SI) models how likely individuals are to pass on the disease after `x` days. A default gamma distribution stats at `0`, but this doesn't make sense under our modelling. As such, our serial intervals must be at least 1 day, so we shift the gamma distribution by 1.

This is also the reason we're doing this discretisation - gamma is continuous but we need a value for 1 day, 2 days, 3 days, etc.

In short, the `discr_si` function exists to adapt the (continuous, starting from 0) gamma distribution into a discrete distribution that starts from 1, which we can then use to model the Serial Interval.

I hope that helps!

**Comment by annecori** [comment]
Hi @tim-laurence-dhsc, 

Thanks for reaching out and showing interest in EpiEstim. I'll try and add to what @jstockwin has already kindly explained. 
In short, discr_si doesn't discretize a gamma distribution in the most naive way, but it accounts for the fact that the serial interval is the difference between two dates, where those two dates are themselves a discretised version of a continuous process. So when you record a SI of 0 for example, this means the two individuals had symptoms on the same day, so the difference between their two continuous times of symptoms can be anything between 0 and 1 day but it's most likely to be around 0.5 days, so it's not uniformly distributed between 0 and 1. Hence the discretisation we've used which accounts for this. 
I would refer you to section 11 of the supplement of our [paper](https://academic.oup.com/aje/article/178/9/1505/89262), which explains this and provides the mathematical derivations. 
Thanks!

**Comment by annecori** [comment]
Suggest @rebeccanash could update the doc with explanation and reference to section 11 of our paper (as per above comment) - with input from me where needed
